### ISSUE TITLE
Update .travis.yml

### ISSUE THREAD

**Issue by BoltzmannBrain** [numenta/NAB#175]
Update this to follow the newest instructions in the README and run against NuPIC 0.3.0:

```
pip install https://s3-us-west-2.amazonaws.com/artifacts.numenta.org/numenta/nupic.core/releases/nupic.bindings/nupic.bindings-0.1.5-cp27-none-linux_x86_64.whl
pip install nupic
```

It's important NAB is runnable w/o NuPIC.

**Comment by SaganBolliger** [comment]
@BoltzmannBrain Has this been fixed by #208?

**Comment by BoltzmannBrain** [comment]
No, we don't want to install nupic with nab.
